File: 03-CPP/08-funciones/02ejercicioSumar.cpp
#include <iostream>
using namespace std;

int sumar(int a, int b)
{
    return a + b;
}

int main()
{

    int a, b;

    cout << "Ingresa el primer valor " << endl;
    cin >> a;

    cout << "Ingresa el segundo valor " << endl;
    cin >> b;
   
    cout << "Suma: " << sumar(a, b) << endl;

    return 0;
}

File: 04_CSHARP/08_Funciones/02ejercicioSumar.cs

// Pedir valores al usuario 
Console.Write("Ingresa el primer valor: ");
var a = Convert.ToInt32(Console.ReadLine());

Console.Write("Ingresa el segundo valor: ");
var b = Convert.ToInt32(Console.ReadLine());

var resultado = Sumar(a, b);
Console.WriteLine($"Suma o total: {resultado}");

// Definir la funcion sumar
int Sumar(int a, int b)
{
    int resultado = a + b;
    return resultado;
}
